src/core/application.hpp

```diff
@@ -1,31 +1,31 @@
 // SPDX-License-Identifier: Apache-2.0
 // Copyright (C) 2021 YADRO
 
-#ifndef BMC_APPLICATION_HPP
-#define BMC_APPLICATION_HPP
+#pragma once
 
 #include <fastcgi++/manager.hpp>
-#include <logger/logger.hpp>
-#include <core/broker/dbus_broker.hpp>
-#include <core/entity/entity.hpp>
-
+#include <phosphor-logging/log.hpp>
+#include <core/connection.hpp>
+#include <core/connect/dbus_connect.hpp>
+#include <core/entity/enitty_manager.hpp>
```

Comment:
The header has a wrong filename: `enitty_manager` -> `entity_manager` ;)
The all `#include` instructions should be fixed too.